dask/dataframe/indexing.py

```diff
@@ -93,10 +93,15 @@ def __getitem__(self, key):
 
     def _loc(self, iindexer, cindexer):
         """ Helper function for the .loc accessor """
+        meta = self._make_meta(iindexer, cindexer)
```

Comment:
I might move this down below the `Series` and `Array` checks, just since it's extra unneeded work. Not necessary though, should be fine here as well.

Reply:
i just didnt want to duplicate this line. it already ads a duplicated return line, i look into cleaning those up